PROTOCOL.md

```diff
@@ -965,6 +966,24 @@ class SetDirtyStateAction implements Action {
 ```
 </details>
 
+### 2.6.3. ExportSvgAction
+
+The client (or the server) sends a `ExportSvgAction` to indicate that diagram, which represents the current model state, should be exported in SVG format. The action provides the diagram svg as plain string.
+
+<details open><summary>Code</summary>
+
+```typescript
+class ExportSvgAction implements ResponseAction {
```

Comment:
Please add the following comment the action to indicate that it stems from Sprotty:
```
/**
 * Sprotty's ExportSvgAction.
 */
```